src/test/app/forms/models/partyDetails.ts

```diff
@@ -19,15 +19,15 @@ describe('PartyDetails', () => {
         line1: 'first line',
         postcode: 'bb127nq'
       },
-      hasCorrespondenceAddress: true,
+      hasCorrespondenceAddress: 'true',
```

Comment:
If input is supposed to mock what is being store in draft store, then that change is wrong. Data in draft store similarly to data in claim store is supposed to have correct type. Revert.

Reply:
Done